extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -159,6 +159,17 @@ export class AmpSlideScroll extends BaseSlides {
       this.slidesContainer_.addEventListener(
           'touchend', this.touchEndHandler_.bind(this));
     }
+
+    this.registerAction('moveToSlideIndex', invocation => {
```

Comment:
Nit: How about `changeSlide`? Either way, probably should be consistent with the new event in #6797.